CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+### 2.31.12
+This release includes:
+* Fluent Bit [1.9.10](https://fluentbit.io/announcements/v1.9.10/)
+* Amazon CloudWatch Logs for Fluent Bit 1.9.3
```

Comment:
Looks like below the new version is `1.9.4` and not `1.9.3`

Reply:
Oops thanks!!!

Reply:
fixed